application/gqlgen/mutations.go

```diff
@@ -245,17 +248,52 @@ func (r *mutationResolver) RemoveOrganizationTrust(ctx context.Context, input Re
 	return &organization, nil
 }
 
-func (r *mutationResolver) CreateMeetingInvitations(ctx context.Context, input CreateMeetingInvitationsInput) ([]MeetingInvitation, error) {
-	return []MeetingInvitation{}, nil
+// CreateMeetingInvites implements the `createMeetingInvites` mutation
+func (r *mutationResolver) CreateMeetingInvites(ctx context.Context, input CreateMeetingInvitesInput) (
+	[]models.MeetingInvite, error) {
+
+	cUser := models.GetCurrentUserFromGqlContext(ctx)
+	extras := map[string]interface{}{
+		"user": cUser.UUID,
+	}
+
+	var m models.Meeting
+	if err := m.FindByUUID(input.MeetingID); err != nil {
+		return nil, reportError(ctx, err, "CreateMeetingInvite.FindMeeting", extras)
+	}
+
+	inv := models.MeetingInvite{
+		MeetingID: m.ID,
+		InviterID: cUser.ID,
+	}
+
+	badEmails := make([]string, 0)
+	for _, email := range input.Emails {
+		inv.Email = email
+		if err := inv.Create(); err != nil {
+			badEmails = append(badEmails, email)
+			domain.ErrLogger.Printf("error creating meeting invite for email '%s', %s", email, err)
```

Comment:
You don't need to change this - I'm just wondering if it would be better to have one log message with all the bad emails together (down in the next `if` block).

Reply:
I thought about that, but we would lose the actual failure message.